src/components/Accordion/AccordionTitle.tsx

```diff
@@ -65,15 +66,15 @@ class AccordionTitle extends UIComponent<ReactProps<AccordionTitleProps>, any> {
     const indicatorWithDefaults = indicator === undefined ? {} : indicator
 
     const contentElement = (
-      <>
-        {Indicator.create(indicatorWithDefaults, {
+      <Layout
+        start={Indicator.create(indicatorWithDefaults, {
```

Comment:
Maybe is worth adding example so the screens will catch this?

Reply:
yes, was thinking about the same thing, was waiting for the same opinions on that :) will add, thank you!